x-pack/metricbeat/module/aws/utils.go

```diff
@@ -44,7 +44,8 @@ func GetListMetricsOutput(namespace string, regionName string, svcCloudwatch clo
 	var nextToken *string
 
 	listMetricsInput := &cloudwatch.ListMetricsInput{
-		NextToken: nextToken,
+		NextToken:      nextToken,
+		RecentlyActive: "PT3H",
```

Comment:
Can we add a comment that explains where this value comes from?

Reply:
did we clarify the behavior of this if the collection period is configured >3h?